code/processes/wdb.q

```diff
@@ -256,7 +256,7 @@ doreload:{[pt]
     reloadcomplete::0b;
     /-inform gateway of reload start
     informgateway(`reloadstart;`);
-    getprocs[;pt] each reloadorder;
+    getprocs[;pt] each reloadorder where reloadorder in .servers.SERVERS[`proctype];
```

Comment:
```suggestion
    getprocs[;pt] each reloadorder inter .servers.SERVERS[`proctype];
```

🏌️

Reply:
Sorry didn't notice this - will add here!